pkg/kubelet/util/csr/csr.go

```diff
@@ -50,30 +55,46 @@ func RequestNodeCertificate(client certificatesclient.CertificateSigningRequestI
 	if err != nil {
 		return nil, fmt.Errorf("unable to generate certificate request: %v", err)
 	}
-	return RequestCertificate(client, csrData, []certificates.KeyUsage{
+	hash := sha512.New512_256()
+	hash.Write(privateKeyData)
+	hash.Write([]byte(subject.CommonName))
+	for _, org := range subject.Organization {
+		hash.Write([]byte(org))
+	}
```

Comment:
And a comment about how this hash should be kept up to date to include everything in the CSR, but can't be a hash of the csrData because of the random elements.